torchx/schedulers/docker_scheduler.py

```diff
@@ -287,6 +288,7 @@ def _submit_dryrun(self, app: AppDef, cfg: DockerOpts) -> AppDryRunInfo[DockerJo
                             LABEL_REPLICA_ID: str(replica_id),
                         },
                         "hostname": name,
+                        "privileged": cfg["privileged"],
```

Comment:
can we change this to be `cfg.get("privileged", False)` for safety?

Reply:
Sure, I'll fix that. Just FYI: https://github.com/pytorch/torchx/blob/a2423f4cf9b0a7b3ddad624d8b06ebddf3b8a3cb/torchx/schedulers/aws_batch_scheduler.py#L512